app/src/components/v-table/table-header/table-header.vue

```diff
@@ -114,6 +116,8 @@ export default defineComponent({
 		useEventListener(window, 'pointermove', throttle(onMouseMove, 40));
 		useEventListener(window, 'pointerup', onMouseUp);
 
+		const showSelect = ['one', 'multiple'].includes(props.selectionMode);
```

Comment:
This should probably be a computed property in case the prop changes.

This also applies to a few cases below.